sys/conf/ldscript.mips

```diff
@@ -51,6 +51,10 @@ SECTIONS
   {
     KEEP (*(.fini))
   } =0x1000000
+  .fill1     :
```

Comment:
What is this?

Reply:
It was a workaround to pad the FPGA image for the DE4. The correct thing to do is to generate a binary image for berictl to load with `objcopy`, but we have not been doing so. Perhaps this should not be in dev and I should have cheribuild use objcopy. I remember there was a reason I did not do that in the first place, but can't entirely recall it now.

Reply:
But why does this only happen for a purecap kernel? Hybrid kernel work just fine.